src/main/resources/assets/chattriggers/js/moduleProvidedLibs.js

```diff
@@ -151,7 +135,7 @@
     loadClass("org.lwjgl.opengl.GL44");
     loadClass("org.lwjgl.opengl.GL45");
 
-    loadClass("com.chattriggers.ctjs.minecraft.listeners.CancellableEvent");
+    const CancellableEvent = Java.class("com.chattriggers.ctjs.api.triggers.CancellableEvent");
```

Comment:
Wasn't this originally added so that custom triggers could pass events?

Reply:
Ahh right that makes sense, I'll revert this then